tests/sentry/utils/audit/tests.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+from django.contrib.auth.models import AnonymousUser
+
+from sentry.models import ApiKey
+from sentry.testutils import APITestCase
+from sentry.utils.audit import create_audit_entry
+
+
+class FakeHttpRequest(object):
+    def __init__(self, user, META):
+        self.user = user
+        self.META = META
+
+
+class CreateAuditEntryTest(APITestCase):
+
+    def test_audit_entry_api(self):
+        org = self.create_organization()
+        apikey = ApiKey.objects.create(
+            organization=org,
+            allowed_origins='*',
+        )
+        meta = {
+            'REMOTE_ADDR': '127.0.0.1',
+            'HTTP_AUTHORIZATION': apikey
```

Comment:
This isn't right. I get what you're trying to do, but this is attempting to test the wrong thing. This `meta` stuff maps to HTTP headers, so in the case here, you're using a Model instance of `ApiKey` which would never be the value of an HTTP header.

The test technically works, but really you shouldn't pass `HTTP_AUTHORIZATION` here, and just assert on `entry.actor_key == apikey`.

Reply:
@mattrobenolt Thanks for the catch. That makes sense. I was wondering how best to test it? Or is it better to simply leave testing the apikey out?